### DISCUSSION TITLE
[End User] Multiple .xpt Dataset - Not moving from 99%

### DISCUSSION THREAD

**Discussion by tapswi2696** [cdisc-org/cdisc-rules-engine#461] [Q&A]
Dear Core Team, 

I am executing the command as following in order to execute all .xpt files present in to directory. For some reasons, rule engine not moving forward from 99% and report also not executed (waited almost 35-40 minutes). 

Command I am executing as follows: 
python3.9 core.py validate -s sdtmig –v 3-4 -d /path/xpt_files/                         

I am missing anything or not using correct command? I added all required parameters such as -s, -v, and -d.

**Comment by SFJohnson24** [comment]
Hi @tapswi2696 -- have you had this issue with recent releases?  If so, I can reopen this issue.  I apologize for how long it took to get a response.  Your question was from before my time at CDISC.  I am happy to help you troubleshoot if this is still an issue.
Best,
Sam
